doc/_resource_types/cloudwatch_logs.md

```diff
@@ -38,3 +38,11 @@ describe cloudwatch_logs('my-cloudwatch-logs-group') do
   end
 end
 ```
+
+### have_tag
+
+```ruby
+describe cloudwatch_logsl('my-cloudwatch-logs-group') do
```

Comment:
`s/cloudwatch_logsl/cloudwatch_logs/` ? Could you fix it ?

Reply:
ah yep